Integrations/integration-MISP_V2.yml

```diff
@@ -1642,7 +1678,23 @@ script:
       description: All tag names in the event
       type: string
     description: Adds a tag to the given UUID event or attribute.
+  - name: misp-add-sighting
```

Comment:
Why are there no outputs for the command?
I think there should be the same as there are for other commands that have attributes outputs

Reply:
add sighting is returning nothing (as well as search, it's returning nothing related to sightings).
There's a problem with MISP.search_sighting, might be related to our instance.

They fixed their documentation after opening an issue, but still there's no response from out instance (tried with postman also). 
https://github.com/MISP/PyMISP/issues/336

Reply:
You don't need anything from the response, even though it would be nice to get something.
Build the context object from the arguments you get.

Reply:
sighting is an increased number, so adding something to the output will be misleading and will include only 1 sighting.